libs/feature-flags/src/lib/features.ts

```diff
@@ -25,6 +25,7 @@ export enum Features {
   complaintsToAlthingiOmbudsman = 'isComplaintToAlthingiOmbudsmanEnabled',
   healthcareLicenseCertificate = 'isHealthcareLicenseCertificateEnabled',
   transferOfMachineOwnership = 'isTransferOfMachineOwnershipEnabled',
+  additionalSupportForTheElderlyApplication = 'isAdditionalSupportForTheElderlyEnable',
```

Comment:
~ Up to you if you want to add the 'd' at the end 😅
```suggestion
  additionalSupportForTheElderlyApplication = 'isAdditionalSupportForTheElderlyEnabled',
```